### PR TITLE
Introduction To C programming

### PR DESCRIPTION
**Pull request by MouliDev**
### Article checklist

**ATTENTION: In your PR - add the ISSUE (topic form) # that this PR corresponds to. You can do this by also adding a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.**
      
      
**NOTE: (Please ensure that you have only one open issue + linked pull request at a time. This will ensure that we complete the article in a timely manner from inception to publishing.)**

**If you have not already please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve our overall technical writing.**

#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker? We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**

### CODE REVIEW

**Comment by hectorkambow** [comment]
[Introduction To C programming #2608.pdf](https://github.com/section-engineering-education/engineering-education/files/6697552/Introduction.To.C.programming.2608.pdf)

👋 Good afternoon - and welcome to the EngEd program.
I'll be running a final review on your PR when ready.
Upon running your article through our 3rd party plagiarism checker it seemed to raise a few flags and the % was higher than we typically accept.
Please see attached PDF - and revisit the article to ensure we are contributing wholly unique and original content.
Be sure to see our resources page to see more info on plagiarism and what is considered as such.

**Comment by MouliDev** [comment]
@hectorkambow 
Hello Sir, Just checked my article I'm sorry for that and I won't repeat it again  and I updated but some commands and names I can't because they are the same every where
Thank You

File: content/articles/introduction-to-c/index.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+- Writing the First C program
+- Files used in C program
+- Uses Of C programming
+
+### History of C Program
+
+C is a practical programming language. In the 1970's "C" was developed by Dennis Ritchie and used by the UNIX operating system at Bell Laboratories.
+
+- Why this language named C?
+  - Before "C" there is a language called "B" and Some of the Options are derived from "B"
```
Comment (WanjaMIKE) [inline content/articles/introduction-to-c/index.md:40]: Could you reference that `B` language.

File: content/articles/introduction-to-c/index.md [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+{
+ Local declarations
+ Statements
+ }
+ Function N()
+{
+ Local declarations
+ Statements
+ }
+```
```
Comment (WanjaMIKE) [inline content/articles/introduction-to-c/index.md:68]: Could you explain the code above

File: content/articles/introduction-to-c/index.md [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+
+- preprocessor directives
+
+  - Preprocessor directives are lines included in the code of programs preceded by a hash sign `#`
+  - One of the most important and commonly used preprocessor commands is `include` which tells the compiler to execute the program
+
+- Functions in C
+  - `C` program contains one or more functions.
+  - A group of `C` statements and that statements executed together is known as Function
```
Comment (WanjaMIKE) [inline content/articles/introduction-to-c/index.md:77]: Rephrase the above sentence

File: content/articles/introduction-to-c/index.md [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+  
+
+  ```
+      #include <stdio.h>
+       int main()
+       {
+        printf("Welcome to Introduction to C programming");
+        return 0;
+       }
+  ```
```
Comment (WanjaMIKE) [inline content/articles/introduction-to-c/index.md:113]: Explain what the above code snippet does

File: content/articles/introduction-to-c/index.md [hunk 5]
```diff
@@ -0,0 +1,202 @@
...
+      - Files contain the set of predefined standard library functions
+      - Extension is `.h`
+    - Object File
+      - These are generated by the compiler 
+      - Extension is `.obj`
+    - Executable File
+      - Executable file is also known as `Binary Executable Files`
+      - Extension is `.exe`
+
+### Uses Of C programming
```
Comment (WanjaMIKE) [inline content/articles/introduction-to-c/index.md:191]: Please add more content to this section

File: content/articles/introduction-to-c/index.md [hunk 6]
```diff
@@ -0,0 +1,202 @@
...
+       int main()
+       {
+        printf("Welcome to Introduction to C programming");
+        return 0;
+       }
+  ```
+
+  - Navigate `File > Save As` to save your file. The select location you want to save and save the source file (the file which you written the code)by changing file type as `C` with the extension of `(*.c)`.
+
+  #### To Run the code
```
Comment (WanjaMIKE) [inline content/articles/introduction-to-c/index.md:117]: Remove extra space before the title. Check the entire article for formatting issues

**Review by WanjaMIKE — CHANGES_REQUESTED** [review]
@MouliDev Hey, could you please address the above suggestions before we continue. Thanks

File: content/articles/introduction-to-c/index.md (hunk 1 above)
Comment (MouliDev) [reply content/articles/introduction-to-c/index.md:40]: added

File: content/articles/introduction-to-c/index.md (hunk 2 above)
Comment (MouliDev) [reply content/articles/introduction-to-c/index.md:69]: And that was not the code it is just a Structure

File: content/articles/introduction-to-c/index.md (hunk 3 above)
Comment (MouliDev) [reply content/articles/introduction-to-c/index.md]: done

File: content/articles/introduction-to-c/index.md (hunk 6 above)
Comment (MouliDev) [reply content/articles/introduction-to-c/index.md:119]: done

File: content/articles/introduction-to-c/index.md (hunk 5 above)
Comment (MouliDev) [reply content/articles/introduction-to-c/index.md:196]: added

**Comment by MouliDev** [comment]
@WanjaMIKE Hey i already updated yesterday only can you please check it out

**Comment by WanjaMIKE** [comment]
Okay, I will check them @MouliDev

**Comment by WanjaMIKE** [comment]
Why did you close this issue?

**Pull request closed without merging**